sender_request.go

```diff
@@ -34,39 +35,48 @@ type SenderRequest struct {
 //
 // Source: https://github.com/moneybutton/paymail-client/blob/master/src/VerifiableMessage.js
 // Specs: http://bsvalias.org/04-01-basic-address-resolution.html#signature-field
-func (s *SenderRequest) Verify(keyAddress, signature string) error {
-
+func (s *SenderRequest) Verify(keyAddress string, signature string) error {
 	// Basic checks before trying the signature verification
 	if len(keyAddress) == 0 {
 		return fmt.Errorf("missing key address")
 	} else if len(signature) == 0 {
 		return fmt.Errorf("missing a signature to verify")
 	}
 
+	data := fmt.Sprintf("%s%d%s%s", s.SenderHandle, s.Amount, s.Dt, s.Purpose)
```

Comment:
This line could be moved to separate functions because I can see it duplicated in `Verify` and in `Sign` method.

Reply:
Sure 👍 done - please check if it's ok